asf_installer_survey/notebooks/section_6_analysis.py

```diff
@@ -0,0 +1,1670 @@
+df_608_crosstab_owner_V2.to_csv("../../outputs/section6/csv/recoded/608_owner_V2_recoded.csv")
+df_608_crosstab_owner_V2
+
+# %%
+# Generate figure
+groupedbar(df_608_crosstab_owner_V2,
+           "608_owner_V2_recoded",
+           "SQ 45. Have you found any challenges working in heat pump repairs, servicing and maintenance?")
+
+# %%
+# Generate figure
+donut(df_608_crosstab_owner_V2,
+           "608_owner_V2_recoded",
+           "SQ 45. Have you found any challenges working in heat pump repairs, servicing and maintenance?")
+
+# %% [markdown]
+# ### RQ 6.09 | Do survey participants who currently work in repairs,servicing and maintenance expect to expand this activity?
+# - SQ: 46
+# - Sample:
+# - Standard_output
+
+# %%
+# Print possible answers
+data[col.q46].value_counts()
+
+# %%
+# Change all Not asked to None
+data.loc[data[col.q46] == "Not asked", col.q46] = None
+
+# Specifying desired order for outputs
+ans_46 = ["Yes",
+          "No",
+          "Don't know",
+          "Total"
+          ]
+
+# Generate crosstab dataframe
+df_609_crosstab = crosstable("EmploymentType",
+                             data,
+                             "46. Do you expect to expand your work in heat pump repairs, servicing and maintenance?",
+                             ans_46
+                             )
+df_609_crosstab = df_609_crosstab.fillna(0)
+df_609_crosstab = df_609_crosstab.astype(int)
+
+# Remove Employee index row because they are not asked
+df_609_crosstab = df_609_crosstab.drop(index='Employee')
+
+# Save to .csv and display
+df_609_crosstab.to_csv("../../outputs/section6/csv/609.csv")
+df_609_crosstab
+
+# %%
+# Generate figure
+groupedbar(df_609_crosstab,
+           "609",
+           "SQ 46. Do you expect to expand your work in heat pump repairs, servicing and maintenance?")
+
+# %%
+# Generate figure
+donut(df_609_crosstab,
+           "609",
+           "SQ 46. Do you expect to expand your work in heat pump repairs, servicing and maintenance?")
+
+# %% [markdown]
+# ### RQ 6.10 | How do survey participant expectations of expanding their work in repairs, servicing and maintenance differ depending on the size of the company they own?
+# - SQ: 46
+# - Sample:
+# - Company_size_owner
+
+# %%
+# Generate crosstab dataframe
+df_610_crosstab_owner_V2 = crosstable("CompanySizeOwnerV2",
+                                      data,
+                                     "46. Do you expect to expand your work in heat pump repairs, servicing and maintenance?",
+                                     ans_46
+                                     )
+
+# Save to .csv and display
+df_610_crosstab_owner_V2.to_csv("../../outputs/section6/csv/610_owner_V2.csv")
+df_610_crosstab_owner_V2
+
+# %%
+# Generate figure
+groupedbar(df_610_crosstab_owner_V2,
+           "610_owner_V2",
+           "SQ 46. Do you expect to expand your work in heat pump repairs, servicing and maintenance?")
+
+# %%
+# Generate figure
+donut(df_610_crosstab_owner_V2,
```

Comment:
Probably want to differentiate this from the previous figures by having some label on the plot stating that it's about the owners. Otherwise can't tell the difference between this one and the previous donut plot.